MantidQt/CustomInterfaces/src/Reflectometry/ReflMainWindowPresenter.cpp

```diff
@@ -20,9 +20,6 @@ ReflMainWindowPresenter::ReflMainWindowPresenter(
   // Tell the tab presenters that this is going to be the main presenter
   m_runsPresenter->acceptMainPresenter(this);
   m_settingsPresenter->acceptMainPresenter(this);
-
-  // Trigger the setting of the current instrument name in settings tab
-  m_runsPresenter->notify(IReflRunsTabPresenter::InstrumentChangedFlag);
```

Comment:
Why was this removed? My understanding is that this notification was triggering the `InstrumentChangedFlag` that would cause `PolarizationCorrections` to be enable/disable depending on the selected instrument.

Without this, if my default instrument is for example `INTER`, when I open the interface, even if `INTER` is the selected instrument in the `Runs` tab, in the `Settings` tab `PolarizationCorrections` is not greyed out. I think what you had here was correct.